apps/vessel-history/features/risk-indicator/risk-indicator.slice.ts

```diff
@@ -35,37 +35,49 @@ export const parseMergedVesselsUniqueId = (id: string): FetchIds[] =>
     const [datasetId, vesselId, tmtId] = x.split('|')
     return { datasetId, vesselId, tmtId }
   })
+
+type ThunkParameters = {
+  idData: FetchIds[]
+  indicator: string
```

Comment:
The indicator could be an enum  including these values: `'encounter', 'fishing', 'port-visit', 'gap', 'vessel-identity', 'coverage'`, WDYT?

Reply:
ok